aspnet/getting-started/installing-on-mac.rst

```diff
@@ -41,24 +47,25 @@ The .NET Execution Environment (DNX) is used to build and run .NET projects. Use
 
 **To install DNX for .NET Core:**
 
-1. Use DNVM to install DNX for .NET Core::
+1. Use DNVM to install DNX for .NET Core:
 
-    dnvm upgrade -r coreclr
+	``dnvm upgrade -r coreclr``
 
 **To install DNX for Mono:**
 
 1. Install `Mono <http://www.mono-project.com/docs/getting-started/install/mac/>`__ for OS X. Alternatively you can install Mono via `Homebrew <http://brew.sh/>`__.
 
-2. Use DNVM to install DNX for Mono::
```

Comment:
The 2nd colon means that the following indented text is preformatted, so you don't have to add the double back ticks.